convert_to_webp.sh

```diff
@@ -0,0 +1,3 @@
+for file in src/assets/images/*.{png,jpg}; do
```

Comment:
Do we need this script? @saadpasta @kartik918

Reply:
Seems like overkill and is compression something we want to "offer" to users. Making our images compressed seems like enough for the PR. Shouldn't add stuff that might confuse people.

Reply:
@aashutoshrathi So can you remove the script.

Reply:
Okay